### PR TITLE
base_contact Spanish translation.

### PR DESCRIPTION
**Pull request by yajo**

### CODE REVIEW

File: base_contact/i18n/es.po [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Country"
+msgstr "País"
+
+#. module: base_contact
+#: field:res.partner,birthdate_date:0
+msgid "Birthdate"
+msgstr "Fecha de nadimiento"
```
Comment (dreispt) [inline base_contact/i18n/es.po]: `nacimiento`?

**Comment by pedrobaeza** [comment]
:+1:

File: base_contact/i18n/es.po (hunk 1 above)
Comment (yajo) [inline base_contact/i18n/es.po]: Thanks, fixed.

**Comment by yvaucher** [comment]
Technically :+1:

**Comment by tafaRU** [comment]
:+1:

**Pull request merged**